lib/appoptics_apm/inst/faraday.rb

```diff
@@ -10,59 +10,52 @@ def self.included(klass)
 
       def run_request_with_appoptics(method, url, body, headers, &block)
         blacklisted = url_blacklisted?
+        remote_call = remote_call?
```

Comment:
probably just me but i found this confusing - a local variable with the same name as a method.

Reply:
For rubyists it should be fine,  in Ruby by convention something ending with a `?` is a method returning a boolean.